### PR TITLE
updated dependencies

### PR DESCRIPTION
**Pull request by jimmywarting**
fyi, tried updating set-value to v4 also but it had a breaking change...
otherwise the test runs just fine

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,9 +29,9 @@
     "get-value": "^3.0.1",
     "has-own-deep": "^1.1.0",
     "kind-of": "^6.0.2",
-    "set-value": "^3.0.0",
-    "union-value": "^1.0.0",
-    "unset-value": "^1.0.0"
+    "set-value": "^3.0.2",
```
Comment (brunoscit) [inline package.json]: can you please update to 4.0.1, there is a high security vulnerability with that version.

https://github.com/advisories/GHSA-4jqc-8m5r-9rpr

Thank you!

P.S: I saw your comment that you can't use version 4 but we can't use this module without this update =[

**Review by brunoscit — CHANGES_REQUESTED** [review]

**Comment by brunoscit** [comment]
> fyi, tried updating set-value to v4 also but it had a breaking change... otherwise the test runs just fine

We can't use it in a version less than 4.0.1, there is a high security issue =[ 
https://github.com/advisories/GHSA-4jqc-8m5r-9rpr

File: package.json (hunk 1 above)
Comment (jimmywarting) [reply package.json]: bumped to 4.1.0

**Comment by jonschlinkert** [comment]
resolved by https://github.com/jonschlinkert/cache-base/pull/23. I do appreciate the PR, thank you.

**Pull request closed without merging**